src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double firstSide;
+    private double secondSide;
+
+    public Rectangle(String color, double firstSide, double secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    public double getFirstSide() {
+        return firstSide;
+    }
+
+    public void setFirstSide(double firstSide) {
+        this.firstSide = firstSide;
+    }
+
+    public double getSecondSide() {
+        return secondSide;
+    }
+
+    public void setSecondSide(double secondSide) {
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double areaCalculation() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle"
+                + ", firstSide = "
+                + firstSide
+                + " units"
+                + ", secondSide = "
+                + secondSide
+                + " units"
+                + ", area = "
+                + areaCalculation()
+                + " units"
+                + ", color = "
+                + getColor()
+        );
```

Comment:
Please make your toString methods shorter. They can be regrouped by field
```suggestion
        System.out.println("Figure: rectangle"
                + ", firstSide = " + firstSide + " units"
                + ", secondSide = " + secondSide + " units"
                + ", area = " + areaCalculation() + " units"
                + ", color = " + getColor());
```
Please recheck all of them